dvc/updater.py

```diff
@@ -164,23 +179,29 @@ def _get_linux(self):
         return package_managers.get(distro.id())
 
     def _get_darwin(self):
-        if not is_binary():
-            if __file__.startswith("/usr/local/Cellar"):
-                return "formula"
-            else:
-                return "pip"
+        if is_binary():
+            return None
 
-        # NOTE: both pkg and cask put dvc binary into /usr/local/bin,
-        # so in order to know which method of installation was used,
-        # we need to actually call `brew cask`
-        ret = os.system("brew cask ls dvc")
```

Comment:
Removed since `dvc` is part of brew now. Brew installation should be checked now in different manner as @shcheklein said

Reply:
thanks, we can remove https://github.com/iterative/dvc/blob/82b69a5abd5278af24b0817092a35ea24f936e4b/dvc/updater.py#L126, right? and may be some cask tests. It's not related to this PR, but since you have started doing this :) Up to you.